### ISSUE TITLE
Add support for repeated, nested sets of questions

### ISSUE THREAD

**Issue by hmpf** [hmpf/easydmp#81]
In order to support RDA DMP Common Standard it is necessary to be able to ask a set of questions multiple times, for instance for `dataset` and `dataset:distribution`.

**Comment by hmpf** [comment]
Have a flag on a section showing whether it can be repeated. Then a way to add a human-friendly id for a copy of a section is needed. Since the data for a section will be in AnswerSet, which has a foreign key to Section, put the id there?

Then if the Section is repeatable, have a way to enter an id for the data of that section.

**Comment by adilhasan** [comment]
The id makes sense.

**Comment by hmpf** [comment]
This is a big, difficult rewrite since we're basically removing "data" from plan and splitting it up into multiple json blobs. We could also have an Answer-model, that hangs off an AnswerSet, that holds the json-blob (though "notes" could be a field), but then we increase the complexity quite a bit in for instance the HTML exporter.

**Comment by hmpf** [comment]
> The id makes sense.

It's an explicit field in the RDA DMP CS. We cold also auto-generate it.

**Comment by hmpf** [comment]
I'll split the remainder into multiple issues.

Correct forwards and backwards traversal when answering questions, for instance: skipping subsections of optional sections when the parent section has been skipped depends on #230 and will be handled by that one.

**Comment by hmpf** [comment]
#238 covers setting the identifier of an answerset, for SECTION_id-fields in RDA DMP CS

**Comment by hmpf** [comment]
One final new issue needed:

Update the UX to work with nested/repeated sections

**Comment by hmpf** [comment]
Closing this issue so that it does not turn into a zombie.
